README.md

```diff
@@ -5,13 +5,19 @@ The CiviAwards extension depends on (uk.co.compucorp.civicase) extension and pro
 and processes.
 
 ## How do I get CiviAwards?
-CiviAwards is designed to work with CiviCRM 5.x plus. If you are on an earlier version of CiviCRM, you will need to upgrade your site first or contact info@compucorp.co.uk if you needs assistance to do so.
 
-If your CiviCRM is already on CiviCRM 5.x plus and this is the first time you use an extension,  please see [Here](http://wiki.civicrm.org/confluence/display/CRMDOC/Extensions "CiviCRM Extensions Installation") for full instructions and information on how to set and configure extensions.
+### Technical Requirements
```

Comment:
Should this heading be Dependencies?

Reply:
I was checking civicase, and there it says `Technical Requirements`, so kept it consistent.